### PR TITLE
Update application.yml.example

### PR DESCRIPTION
**Pull request by davidwisniowski**
i still need the ?createDatabaseIfNotExist=true parameter. without this parameter it says at starting spring boot: unknown database even with hibernate ddl-auto create-drop

### CODE REVIEW

**Comment by victorwrotethis** [comment]
Good point. If you could be so kind to change the create-drop to update instead in the same PR that would be great.
While beneficial for testing it is something I disable on production or otherwise on a restart you lose your data.

**Comment by davidwisniowski** [comment]
done :)

File: src/main/resources/application.yml.example [hunk 1]
```diff
@@ -14,6 +14,6 @@ spring:
     password: example
   jpa:
     hibernate:
-      ddl-auto: create-drop
+      ddl-auto: create-update
```
Comment (victorwrotethis) [inline src/main/resources/application.yml.example]: Close! it's update only, omit the `create-`. For context, here are the possible options:

```
none: Disable DDL handling.
validate: Validate the schema, make no changes to the database.
update: Update the schema if necessary.
create: Create the schema and destroy previous data.
create-drop: Create and then destroy the schema at the end of the session.
```

Update will also create the schema if there is none as 'it is necessary' as it doesn't match what it expects.
Create is a destructive operation. They definitely could have considered a different model at hibernate, but it's what we have to work with :).

**Review by victorwrotethis — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml.example (hunk 1 above)
Comment (davidwisniowski) [reply src/main/resources/application.yml.example]: oh - damn, sorry. just changed it without testing. i have removed the prefix: create-

**Review by davidwisniowski — COMMENTED** [review]
changed.

**Review by victorwrotethis — APPROVED** [review]

**Pull request merged**